library.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "RF24",
+  "keywords": "rf, radio, wireless, spi",
+  "description": "Optimized High Speed Driver for nRF24L01(+) 2.4GHz Wireless Transceiver",
+  "repository":
+  {
+    "type": "git",
+    "url": "https://github.com/TMRh20/RF24.git"
+  },
+  "include": [
+    "*.h",
+    "*.c",
+    "*.cpp",
+    "arch/ATTiny/*",
```

Comment:
PlatformIO builds all source code from library by default. Additional `.cpp/c` files which don't have relation to current platform (selected by user) will cause build error.
See supported development platforms here: http://platformio.org/#!/platforms